smart-contracts/package.json

```diff
@@ -22,7 +22,7 @@
     "truffle-contract": "^3.0.6",
     "web3-utils": "^1.0.0-beta.36",
     "zos": "^v2.1.0-rc.0",
-    "zos-lib": "^2.0.2"
+    "zos-lib": "^v2.1.0-rc.0"
```

Comment:
cc @nfurfaro Is that the version *without* the heavy dependencies?

Reply:
Yeah, zos-lib is tiny compared to zos. 29M vs 1071M